src/app/_mermaid_common/libs/directives/paginated-table.dir.js

```diff
@@ -722,6 +722,13 @@ angular
             $scope.fetchTableRecords(true);
           };
 
+          $scope.control.searchFilterUsed = function() {
```

Comment:
I think to be clear this should be named something specific to the search textbox, like `$scope.control.textboxFilterUsed`